javascript-data-model/order-history.js

```diff
@@ -0,0 +1,98 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4th, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-111111',
+    viewOrderDetails: false,
+    viewinvoice: false,
+    orderDelivered: 'August 8, 2020',
+    image: 'url',
+    returnWindowClosed: true,
+    subTotal: '$31.55',
+    buyAgain: false,
+    viewItem: false,
+    writeReview: false,
+    handedToResident: false,
+    type: [
+      {
+        itemType: 'book',
+        title: 'JavaScript for impatient programmers',
+        author: 'Dr. Axel Rauschmayer'
+      }
+    ]
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    total: '$44.53',
+    shipTo: 'JS Masher',
+    orderNumber: '113-2222222',
+    viewOrderDetails: false,
+    viewinvoice: false,
+    orderDelivered: 'Jul 12, 2020',
+    image: 'url',
+    returnWindowClosed: true,
+    subTotal: '$41.33',
+    buyAgain: false,
+    viewItem: false,
+    writeReview: false,
+    handedToResident: true,
+    type: [
+      {
+        itemType: 'book',
+        title: 'The Timeless Way of Building',
+        author: 'Christopher Alexander'
+      }
+    ]
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    total: '$17.22',
+    shipTo: 'JS Masher',
+    orderNumber: '115-123456',
+    viewOrderDetails: false,
+    viewinvoice: false,
+    orderDelivered: 'Jul 7, 2020',
+    image: 'url',
+    returnWindowClosed: true,
+    subTotal: '$15.98',
+    buyAgain: false,
+    viewItem: false,
+    writeReview: false,
+    handedToResident: true,
+    type: [
+      {
+        itemType: 'electronic',
+        description: 'Gamecube Controller'
+      }
+    ]
+  },
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138.93',
+    shipTo: 'JS Masher',
+    orderNumber: '116-111111',
+    viewOrderDetails: false,
+    viewinvoice: false,
+    orderDelivered: 'Jul 5, 2020',
+    image: 'url',
+    returnWindowClosed: true,
+    buyAgain: false,
+    viewItem: false,
+    writeReview: false,
+    handedToResident: false,
+    type: [
+      {
+        itemType: 'electronic',
+        subTotal: '$94.95',
+        description: 'Gamecube Controller'
+      },
+      {
+        itemType: 'book',
+        title: 'The Art of Sql',
+        author: 'Stephane, Faroult',
+        subTotal: '$33.99'
+      }
```

Comment:
Each of these items will need a separate image, having one `image` property on the order wouldn't work here.